Maestro.json

```diff
@@ -74,7 +74,8 @@
         {
           "FeederFix": "BOREE",
           "ArrivalRegex": "/BOREE\\d[AP]/",
-          "AircraftType": "Jet",
+          "Category": "Jet",
+          "AdditionalAircraftTypes": ["DH8D"],
```

Comment:
The flow tables in the VATPAC SOPs group Jets and DH8D together. Should this be a global setting, or something configurable per-STAR?

Reply:
That's a good point. This doesn't really account for specific aircraft types. Only Jet or NonJet + some whitelisted types.
I'll add another field for specific types.